tests/test_open_prs_search.py

```
@@ -72,3 +76,76 @@ def test_list_open_pull_requests_for_author_uses_search_and_allowlist(monkeypatc
+        }
+    )
+    adapter._client = client  # type: ignore[assignment]
+    monkeypatch.setattr("ghdcbot.adapters.github.rest._load_repo_filter", lambda: None)
+
+    prs = adapter.list_pull_requests_for_author("alice")
+
+    assert len(client.calls) == 1
+    _method, path, params = client.calls[0]
+    assert path == "/search/issues"
+    assert params is not None
+    assert "is:pr author:alice org:AOSSIE-Org" == params["q"]
```

Ruff (0.16.0)

[warning] Yoda condition detected

Rewrite as `params["q"] == "is:pr author:alice org:AOSSIE-Org"`

(SIM300)